src/Styles.re

```diff
@@ -6,8 +6,16 @@ let backgroundImageGorilla = [%bs.raw
   {|require("../src/img/wildcardsimages/wild-jungle-background.png")|}
 ];
 
-Css.(global("body", [fontFamily("Lato, Helvetica, sans-serif")]));
-Css.(global("a", [textDecoration(none)]));
+Css.(
+  global(
+    "body",
+    [margin(`px(0)), fontFamily("Lato, Helvetica, sans-serif")],
+  )
+);
+Css.(global("a", [color(hex("#303030")), textDecoration(none)]));
+Css.(global("a:hover", [textDecoration(underline)]));
+Css.(global("a:active", [color(hex("#303030"))]));
+Css.(global("a:visited", [color(hex("#303030"))]));
```

Comment:
This is the spot causing the global link colors I think

Reply:
Yeah it needs to be "303030" not "#303030" Fixed!